contracts/interfaces/IStakingPool.sol

```diff
@@ -36,8 +47,8 @@ interface IStakingPool is IERC721 {
   // tranche index is calculated based on the expiration date
   // the initial proposal is to have 4 tranches per year (1 tranche per quarter)
   struct Tranche {
-    uint stakeShares;
-    uint rewardsShares;
+    uint /* uint128 */ stakeShares;
```

Comment:
any reason why type is not yet set to `uint128` already?

Reply:
will be done with packing